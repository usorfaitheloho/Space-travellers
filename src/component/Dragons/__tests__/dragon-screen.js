import React from 'react';
import render from 'react-test-renderer';
import { Provider } from 'react-redux';
import Dragon from '../Dragons';
import store from '../../../redux/configureStore';

test('Dragon screen renders correctly', () => {
  const tree = render.create(
    <Provider store={store}>
      <Dragon />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
