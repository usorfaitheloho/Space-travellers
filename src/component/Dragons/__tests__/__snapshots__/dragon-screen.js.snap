// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dragon screen renders correctly 1`] = `
<div
  className="dragon-container"
/>
`;
